words/words.md

```diff
@@ -358,117 +358,117 @@ But wait... haven't Taiwan and South Korea *already* contained COVID-19? For 4 w
 
 How?
 
-###Scenario 4: Test, Trace, Isolate
+###Scénario 4 : Test, Suivi, Isolement
 
-*"Sure, we \*could've\* done what Taiwan & South Korea did at the start, but it's too late now. We missed the start."*
+*« Bien sûr, on \*pourrait\* avoir fait comme Taïwan et la Corée du Sud ont fait au début, mais c'est désormais trop tard. On a loupé le départ. »*
 
-But that's exactly it! “A lockdown isn't a cure, it's just a restart”... **and a fresh start is what we need.**
+Mais c'est justement ça ! « Un confinement n'est pas un remède, c'est juste un retour à la case départ »… **et c'est exactement d'un nouveau départ dont nous avons besoin.**
 
-To understand how Taiwan & South Korea contained COVID-19, we need to understand the exact timeline of a typical COVID-19 infection[^timeline]:
+Pour comprendre comment Taïwan et la Corée du Sud ont contenu le Covid-19, nous devons comprendre le déroulement précis d'une infection typique au Covid-19[^timeline] :
 
-[^timeline]: **3 days on average to infectiousness:** “Assuming an incubation period distribution of mean 5.2 days from a separate study of early COVID-19 cases, we inferred that infectiousness started from 2.3 days (95% CI, 0.8–3.0 days) before symptom onset” (translation: Assuming symptoms start at 5 days, infectiousness starts 2 days before = Infectiousness starts at 3 days) [He, X., Lau, E.H.Y., Wu, P. et al.](https://www.nature.com/articles/s41591-020-0869-5)  
+[^timeline]: **3 days on average to infectiousness:** “Assuming an incubation period distribution of mean 5.2 days from a separate study of early COVID-19 cases, we inferred that infectiousness started from 2.3 days (95% CI, 0.8–3.0 days) before symptom onset” (translation: Assuming symptoms start at 5 days, infectiousness starts 2 days before = Infectiousness starts at 3 days) [He, X., Lau, E.H.Y., Wu, P. et al.](https://www.nature.com/articles/s41591-020-0869-5) Traduction : **3 jours en moyenne jusqu'à l'infectiosité :** « En présumant une distribution de la période d'incubation avec une moyenne de 5,2 jours grâce à une autre étude des premiers cas de Covid-19, nous avons déduit que l'infectiosité démarrait 2,3 jours (IC 95%, 0,8–3,0 jours) avant l'apparition des symptômes » (traduction : En présumant que les symptômes démarrent à 5 jours, l'infectiosité démarre 2 jours plus tôt = l'nfectiosité démarre à 3 jours)
```

Comment:
la même ressource est citée ailleurs, voilà ce qu'on avait mis: 

[^latent]: “Assuming an incubation period distribution of mean 5.2 days from a separate study of early COVID-19 cases, we inferred that infectiousness started from 2.3 days (95% CI, 0.8–3.0 days) before symptom onset” Traduction : "En supposant que la période d'incubation est en moyenne de 5.2 jours en s'appuyant sur une autre étude des premiers cas de COVID-19, nous en déduisons que la contagiosité commence au bout de 2.3 jours (IC 95% : 0.8–3.0 jours) avant l'apparition des symptômes". (explication : En supposant que les symptômes commencent au 5ème jour, sachant que le caractère infectieux commence 2 jours avant, on devient infectieux au bout de 3 jours) He, X., Lau, E.H.Y., Wu, P. et al.

Reply:
Je croyais que le format était quote authors trad, ici c'est quote trad authors…

Reply:
Attention, `"foo"` ne sont pas des guillemets normaux en typographie française, qui emploit `« foo »`. Et le séparateur de décimale est `,`.